test/unittests/backends/webgl/test_pack_unpack.ts

```diff
@@ -154,6 +154,15 @@ describe('#UnitTest# - pack - Tensor pack', () => {
     inferenceHandler = sessionhandler.createInferenceHandler();
   });
   const testDataSet = getTestData();
+
+  // TODO support WebGl 1.0 to run it on Mac
```

Comment:
use `createWebGLContext()` from webgl-context-factory.ts to perform a webgl version check

Reply:
updated